README.rst

```diff
@@ -53,24 +53,29 @@ The ``on`` property reports and sets the on/off state of the light. For this, we
           'on',
           Value(True, lambda v: print('On-State is now', v)),
           metadata={
+              '@type': 'OnOffProperty', 
+              'label': 'On/Off',
               'type': 'boolean',
               'description': 'Whether the lamp is turned on',
           }))
 
-The ``level`` property reports the brightness level of the light and sets the level. Like before, instead of actually setting the level of a light, we just log the level to std::out.
+The ``brightness`` property reports the brightness level of the light and sets the level. Like before, instead of actually setting the level of a light, we just log the level.
```

Comment:
just a remark, actually I expected the brightness to be expressed in physical units like lx ... 
if not level might be more explicit to user.

My 2c

Reply:
That makes sense. The property can actually be named anything you want. It will map to a `BrightnessProperty` type, though, so I went with `brightness`.

Reply:
Well actually you're right brightness is not illuminance, sorry for missleading.

Note, that OCF also used % in OneIoTa model:
https://www.oneiota.org/revisions/2399 

So it's aligned here.